### ISSUE TITLE
Personal info section of checkout is missing aria-describedby attributes and has improperly escaped translation strings

### ISSUE THREAD

**Issue by Shelob9** [easydigitaldownloads/easy-digital-downloads#5334]

**Comment by pippinsplugins** [comment]
Thanks @Shelob9, we'll get it reviewed after 2.7.

**Comment by Shelob9** [comment]
@pippinsplugins OK. No worries, I can unhook those sections and replace with what I used for these two PRs for now.

**Comment by ghost1227** [comment]
Tested good! Though it does now need to be refreshed.

**Comment by pippinsplugins** [comment]
@ghost1227 could you please merge it into an issue branch and refresh it?

**Comment by ghost1227** [comment]
@pippinsplugins Done

**Comment by pippinsplugins** [comment]
One minor change needed on the PR.

**Comment by SDavisMedia** [comment]
Very nice. All is well for me on `issue/5334`. :+1:

**Comment by mintplugins** [comment]
I noticed the billing information is missing aria-describedby as well. Problem?

**Comment by pippinsplugins** [comment]
@mintplugins yes but let's add it later since this was specific to the personal info section.
